daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -776,7 +776,7 @@ func (devices *DeviceSet) poolHasFreeSpace() error {
 
 	minFreeMetadata := (metadataTotal * uint64(devices.minFreeSpacePercent)) / 100
 	if minFreeMetadata < 1 {
-		minFreeData = 1
```

Comment:
Yes, this is a bug. it should have been minFreeMetadata=1. Will generate a PR to fix it. Thanks for catching this.